backend/mypy.ini

```diff
@@ -0,0 +1,25 @@
+[mypy]
```

Comment:
seems like you have integrated dylan's changes in this PR - try rebase and commit again

Reply:
Nope - Dylan's commits just used the default MyPy settings, I changed a few of them + added support for Django.